lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -92,7 +92,9 @@ def bad_pattern?(description)
         end
 
         def expect_patterns
-          inspected = allowed_patterns.map(&:inspect)
+          inspected = allowed_patterns.map do |pattern|
+            pattern.inspect.gsub(/^"|"$/, '/')
```

Comment:
If the conversion process for display here was not changed, it  displayed as follows.
https://github.com/ydah/rubocop-rspec/blob/c017ffd20d2b78f7fda8b129d1aa85ad80edac70/spec/rubocop/cop/rspec/context_wording_spec.rb#L170-L171

```ruby
          context 'this is an incorrect context' do
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Context description should match "patterns", or /^when\\b/.
````

Reply:
If you see a need to separate commits or add entries to the CHANGELOG.md, please point out.

Reply:
`^`  and `$` are used, do you expect multiline? If not, `\A` and `\z` should be used.

Reply:
Indeed it is, thank you very much. I updated this PR.